src/main/java/com/redhat/jenkins/plugins/bayesian/Bayesian.java

```diff
@@ -104,6 +106,7 @@ public BayesianStepResponse submitStackForAnalysis(Collection<FilePath> manifest
         httpPost.setEntity(multipart);
         httpPost.setHeader("Authorization", "Bearer " + getAuthToken());
         httpPost.setHeader("UserEmail", getEmail());
+        httpPost.setHeader("scanRepoUrl", getGitUrl());
```

Comment:
We use upper camel case notation for `UserEmail`, but `scanRepoUrl` is lower camel case. Why the inconsistency?